store.go

```diff
@@ -240,3 +265,79 @@ func (s *SafeConfigStore) Size() int {
 	defer s.mu.Unlock()
 	return s.c.Size()
 }
+
+// LogableConfigStore implements the ConfigStore interface and provides a store
+// that is safe to read and write from multiple goroutines.
+type LogableConfigStore struct {
```

Comment:
And according to [the internet](https://english.stackexchange.com/questions/106938/why-is-it-loggable-rather-than-logable) this should be `Loggable`, with two g's